build-and-push.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+eval $(crc console --credentials | grep "admin, run" | cut -f2 -d"'") --insecure-skip-tls-verify=true
+
+
+REGISTRY=$(oc registry info)
+TOKEN=$(oc whoami -t)
+INTERNAL_REGISTRY=$(oc registry info --internal=true)
+buildah login --tls-verify=false -u openshift -p "${TOKEN}" "${REGISTRY}"
+buildah bud -f build/Dockerfile -t "${REGISTRY}/service-telemetry/service-telemetry-operator:latest" .
+buildah push --tls-verify=false "${REGISTRY}/service-telemetry/service-telemetry-operator:latest"
+
+oc apply -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: smartgateway-operator
+  namespace: service-telemetry
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: smartgateway-operator
+  source: infrawatch-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: smart-gateway-operator.v2.0.0
+EOF
+
+oc delete servicetelemetry stf-default
```

Comment:
I'm not sure if the building and pushing script should be deleting this.

Reply:
you'd want to remove the old operator and also the deployment in order to test the new operator.

Reply:
You can also use the `remove_stf.sh` which exists in `deploy/`